src/app/components/home/post-detail/post-detail.component.scss

```diff
@@ -0,0 +1,3 @@
+.word-break {
+	word-break: break-all !important;
```

Comment:
Why do you need the !important marker?

Reply:
It was a typo, I've copy-pasted the rule from another file and forgot to remove that marker, thank you!